pydsa/cocktail_sort.py

```diff
@@ -0,0 +1,22 @@
+#cocktail sort
+
+def sort(s):
```

Comment:
Change the method name appropriately and also add doc strings. Please check previously committed algorithms in the repo, on how to do it.